index.html

```diff
@@ -181,59 +317,66 @@ <h4 class="h4-5">HTML</h4>
                 <h4 class="h4-6">Bootstrap</h4>
                 <h4 class="h4-7">Ruby</h4>
             </div>
-            <button class="button-2" type="submit">See Projects</button>
+
         </section>
+    </div>  
         <!-- about me section -->
-        <section class="section-9">
-            <h1 class="about-me">About me</h1>
-            <p class="para-4">Hello I’m a software developer! I can help
-                you build a product, feature or website
-                Look through some of my work and 
-                experience! If you like what you see and
-                have a project you need coded,
-                don’t hesitate to contact me.</p>
-            <button class="button-3" type="submit">Get my resume</button>
-        </section>
+        <div class="section-9-1">
+            <section class="section-9">
+                <h1 class="about-me">About me</h1>
+                <p class="para-4">Hello I’m a software developer! I can help
+                    you build a product, feature or website
+                    Look through some of my work and 
+                    experience! If you like what you see and
+                    have a project you need coded,
+                    don’t hesitate to contact me.</p>
+                <button class="button-3" type="submit">Get my resume</button>
+            </section>
+        
         <!-- background img-2 -->
+        
         <section  class="div-img">
-        <img src="Illustration-2.png" alt="" class="img-3">
-           </section>
-           
+            <img src="images/background-desktop-2.png" alt="" class="background-desktop-2">
```

Comment:
- [ ] You did a great job improving the layout. :+1: Kindly remove this background image from the HTML and set it as a CSS `background-image` property as suggested by the previous reviewer. We are doing this because the image is used for aesthetic purposes only and is not part of the contents of the page. ✍️ Please check this [Stackoverflow discussion](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) to get more information about the topic.